spec/features/recipes/show_recipe_page_spec.rb

```diff
@@ -0,0 +1,18 @@
+feature "user visits show recipe" do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
```

Comment:
Please create second recipe just to make sure that we hit the recipe that we really want. And make our test hit the second recipe.